tests/CommandsTest.php

```diff
@@ -111,28 +111,43 @@
 
 test('dump command works', function () {
     $tenant = Tenant::create();
+    $schemaPath = 'tests/Etc/tenant-schema-test.dump';
+
     Artisan::call('tenants:migrate');
 
-    tenancy()->initialize($tenant);
+    expect($schemaPath)->not()->toBeFile();
 
-    Artisan::call('tenants:dump --path="tests/Etc/tenant-schema-test.dump"');
-    expect('tests/Etc/tenant-schema-test.dump')->toBeFile();
+    Artisan::call('tenants:dump ' . "--tenant='$tenant->id' --path='$schemaPath'");
+
+    expect($schemaPath)->toBeFile();
 });
 
-test('tenant dump file gets created as tenant-schema.dump in the database schema folder by default', function() {
-    config(['tenancy.migration_parameters.--schema-path' => $schemaPath = database_path('schema/tenant-schema.dump')]);
+test('dump command generates dump at the passed path', function() {
+    $tenant = Tenant::create();
+
+    Artisan::call('tenants:migrate');
+
+    expect($schemaPath = 'tests/Etc/tenant-schema-test.dump')->not()->toBeFile();
+
+    Artisan::call("tenants:dump --tenant='$tenant->id' --path='$schemaPath'");
 
+    expect($schemaPath)->toBeFile();
+});
+
+test('tenant dump file gets created at the path specified in the tenancy migration parameters config', function() {
```

Comment:
How does this test assert that?

Reply:
Seems like the change you made undid that. You should be setting the config key, not passing it via `--path`.

Reply:
After this is resolved, double check the diff that everything makes sense and you can squash merge (if GH lets you) since you mentioned that you need this for the testing stuff. If GH doesn't let you squash merge it, you can use the `dev-fix-tenant-dump-command` version constraint locally 👍

Reply:
Yeah, the test doesn't make sense, and it asserts the same things as the previous one, so I'll delete it.

For the record, I updated the command so that if `--path` isn't passed, `config('tenancy.migration_parameters.--schema-path')` is used instead of the hardcoded default. I updated the test and then remembered that [the default was hardcoded on purpose](https://github.com/archtechx/tenancy/pull/985#issuecomment-1290798819), so I undid the changes, but I forgot to delete this test.

Reply:
Merged